source/deployment/sso-saml-onelogin.rst

```diff
@@ -28,23 +28,41 @@ Create a OneLogin Connection App for Mattermost SSO
 
 2. Configure the app.
   a. Select the **Configuration** tab, then enter the following values:
+    - **RelayState**: leave blank
+    - **Audience**: leave blank
     - **Recipient**: ``https://<your-mattermost-url>/login/sso/saml`` where ``https://<your-mattermost-url>`` should typically match the `Mattermost Site URL <https://docs.mattermost.com/administration/config-settings.html#site-url>`__.
     - **ACS (Consumer) URL Validator**: ``https:\/\/<your-mattermost-url>\/login\/sso\/saml``
     - **ACS (Consumer) URL**: ``https://<your-mattermost-url>/login/sso/saml``
 
   .. image:: ../../source/images/onelogin_3_configuration_1.png
+  
+  b. In System Console,  `enable encryption <https://docs.mattermost.com/administration/prev-config-settings.html#enable-encryption>`__, save your changes, then return to the Configuration tab in access the **SAML Encryption** field.
 
-  b. Paste the Public key that you generated earlier into the **SAML Encryption** field.
+  c. Paste the Public Key that you generated earlier into the **SAML Encryption** field.
 
   .. image:: ../../source/images/onelogin_4_configuration_2.png
 
-  c. Select **Save**.
+  d. Select **Save**.
 
-3. Enter the attribute parameters.
+3. Enter the attribute parameters. The following attributes are recommended:
+
++----------------+-------------+
```

Comment:
@icelander - Is Username no longer a required attribute?

Reply:
Good catch! Here's a screenshot from my test instance of OneLogin
![Screen Shot 2021-04-12 at 1 00 37 PM](https://user-images.githubusercontent.com/131287/114432808-14693580-9b8f-11eb-8012-cd1c0488deb4.png)